### DISCUSSION TITLE
Flatpak in the documentation

### DISCUSSION THREAD

**Discussion by sr66139p** [NvChad/NvChad#2799] [General]
NvChad works perfectly in the flatpak version of NeoVim, but because the files are in a different location, it can be tricky to set up.  The current Linux/Macos script does not work to install NvChad on flatpak NeoVim. Instead, you must use this script `git clone https://github.com/NvChad/starter ~/.var/app/io.neovim.nvim/config/nvim && flatpak run io.neovim.nvim`. Can we add the flatpak script to the install guide? You could also just make a note about what the equivalent folders are in flatpak 
`~/.config/nvim` = `~/.var/app/io.neovim.nvim/config/nvim`
`~/.local/share/nvim` = `~/.var/app/io.neovim.nvim/data/nvim`
`~/.local/state/nvim` = `~/.var/app/io.neovim.nvim/.local/state/nvim`

**Comment by siduck** [comment]
@sr66139p will do tomorrow, thanks for letting me know.

**Comment by RaptaG** [comment]
The wiki was never updated.. I have a question, does the NVChad update utility work with Flatpak?

**Reply by siduck** [reply]
i forgot to add. nvchad update relies on lazy sync so it should work

**Reply by CleoMenezesJr** [reply]
> The wiki was never updated.. I have a question, does the NVChad update utility work with Flatpak?

Yes, I maintain Neovim Flatpak and use Nvchad as well

**Reply by siduck** [reply]
@CleoMenezesJr nice! i'll add it in the docs tomorrow

**Comment by siduck** [comment]
@CleoMenezesJr @RaptaG @sr66139p done

https://github.com/NvChad/nvchad.github.io/commit/e2fc4839d704f6f2ec65d62ce81c989ee8437562

**Reply by CleoMenezesJr** [reply]
Lovely!! Thx
